<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imagen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>352</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>626</width>
     <height>352</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>621</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:Transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Copyright Â© 2019, Structure Search Engine. All Rights Reserved.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
